pkg/webui/components/notification/notification.styl

```
@@ -41,5 +43,17 @@ left-border($color)
     left-border($c-active-blue)
     color: $c-active-blue
 
-  span:last-child
+  &.success
+    left-border($c-success)
+    color: $c-success
+
+  .title
+    margin: 0
+    font-weight: 600
+
+  .content
     margin-left: $cs.xs
+    display: inline-block
```

```suggestion
    display: inline-block
    color: $tc-deep-gray
```
The text should be deep gray regardless of success/warning/etc.